src/languages/es.ts

```diff
@@ -1417,38 +1417,38 @@ export default {
             customerServiceDetails: 'No hay tarifas de servicio al cliente.',
             inactivityDetails: 'No hay tarifa de inactividad.',
             sendingFundsTitle: 'Enviar fondos a otro titular de cuenta',
-            sendingFundsDetails: 'No se aplica ningún cargo por enviar fondos a otro titular de cuenta utilizando su saldo cuenta bancaria o tarjeta de débito',
+            sendingFundsDetails: 'No se aplica ningún cargo por enviar fondos a otro titular de cuenta utilizando tu saldo cuenta bancaria o tarjeta de débito',
             electronicFundsStandardDetails:
-                'No hay cargo por transferir fondos desde su billetera Expensify ' +
-                'a su cuenta bancaria utilizando la opción estándar. Esta transferencia generalmente se completa en' +
-                '1-3 negocios días.',
+                'No hay cargo por transferir fondos desde tu billetera Expensify ' +
+                'a tu cuenta bancaria utilizando la opción estándar. Esta transferencia generalmente se completa en' +
+                '1-3 días laborables.',
             electronicFundsInstantDetails:
-                'Hay una tarifa para transferir fondos desde su billetera Expensify a ' +
-                'su tarjeta de débito vinculada utilizando la opción de transferencia instantánea. Esta transferencia ' +
-                'generalmente se completa dentro de varios minutos. La tarifa es el 1.5% del monto de la ' +
-                'transferencia (con una tarifa mínima de $ 0.25). ',
+                'Hay una tarifa para transferir fondos desde tu billetera Expensify a ' +
+                'la tarjeta de débito vinculada utilizando la opción de transferencia instantánea. Esta transferencia ' +
+                'generalmente se completa dentro de varios minutos. La tarifa es el 1,5% del importe de la ' +
+                'transferencia (con una tarifa mínima de US$0,25). ',
             fdicInsuranceBancorp:
-                'Sus fondos son elegibles para el seguro de la FDIC. Sus fondos se mantendrán en o ' +
-                `transferido a ${CONST.WALLET.PROGRAM_ISSUERS.BANCORP_BANK}, una institución asegurada por la FDIC. Una vez allí, sus fondos ` +
-                `están asegurados a $ 250,000 por la FDIC en caso de que ${CONST.WALLET.PROGRAM_ISSUERS.BANCORP_BANK} quiebre. Ver`,
-            fdicInsuranceBancorp2: 'para detalles.',
-            contactExpensifyPayments: `Comuníquese con ${CONST.WALLET.PROGRAM_ISSUERS.EXPENSIFY_PAYMENTS} llamando al + 1833-400-0904, por correoelectrónico a`,
+                'Tus fondos pueden acogerse al seguro de la FDIC. Tus fondos se mantendrán o serán ' +
```

Comment:
NAB but I liked `son elegibles para el seguro` more

Reply:
It was a little off for me so I used https://www.deepl.com/en/translator and it gave me the current suggestion. I feel it sounds better.